packages/v4/src/content/design-guidelines/components/code-block/code-block.md

```diff
@@ -0,0 +1,30 @@
+---
+id: Code block
+section: components
+---
+
+A **code block** is a component that contains two or more lines of ready-only code. The code in a code block can be copied to the clipboard.
+
+## Elements
+Code block has four main components.
+
+<img src="./img/code-block-structure.png" alt="Elements of the code block component" width="441"/>
+
+1. **Code snippet:**  the content that will be copied to the user’s clipboard.
+2. **[Action buttons](#multiple-action-buttons):** completes an action when clicked. The copy button is always present but additional action buttons can be added on an as needed basis.
+3. **Tooltip:** informs users that clicking the button will copy the content to their clipboard, or that the content has successfully been copied.
+4. **[Show more/less (optional)](#show-more/less):** expands and collapses the code snippet.
```

Comment:
Last thing - sorry I didn't know this until now. It seems like you can't use a / in a jump link like that! Could you change it to `#show-moreless`. After that, it'll be good to merge!

Reply:
No problem thanks for being thorough with it. I'll update that right now.

Reply:
@lwrigh There shouldn't be a dash between more and less - should just be "moreless"! :/ Hopefully this will be it for real!

Reply:
Sorry about that, I'll update it.